common/lib/xmodule/xmodule/modulestore/tests/test_publish.py

```diff
@@ -1122,20 +1088,7 @@ def test_delete_sequential(self, modulestore_builder, revision_and_result):
             self.assertOLXIsPublishedOnly(block_list_to_delete)
             self.delete_item(block_list_to_delete, revision=revision)
             self._check_for_item_deletion(block_list_to_delete, result)
-            # MODULESTORE_DIFFERENCE
-            if self.is_split_modulestore:
-                # Split:
-                if revision == ModuleStoreEnum.RevisionOption.published_only:
-                    # If deleting published_only items, the children that are drafts remain.
-                    self.assertOLXIsDraftOnly(block_list_children)
-                else:
-                    self.assertOLXIsDeleted(block_list_children)
-            elif self.is_old_mongo_modulestore:
-                # Old Mongo:
-                # If deleting draft_only or both items, the drafts will be deleted.
-                self.assertOLXIsDeleted(block_list_children)
-            else:
-                raise Exception("Must test either Old Mongo or Split modulestore!")
+            self.assertOLXIsDeleted(block_list_children)
```

Comment:
@doctoryes: A side effect of some of my changes was to eliminate a difference in behavior between split and old-mongo.